### ISSUE TITLE
User defined exception in transaction is not returned

### ISSUE THREAD

**Issue by mpv1989** [arangodb/arangodb#2561]
When executing a transaction which throws a user defined exception, the error response does not contain the exception.

## to reproduce:
```
cat <<-___EOB___ | curl -X POST --dump - -u root: -d @- 'http://127.0.0.1:8529/_api/transaction'
{"action":"function (params) {throw 'My error context';}","collections":{}}
___EOB___
```
## result
`{"error":true,"code":500,"errorNum":500,"errorMessage":"internal server error"}`
## expected result (like in 3.1)
`{"exception":"My error context","error":true,"code":500,"errorNum":500,"errorMessage":"internal server error"}`

## my environment running ArangoDB
I'm using the latest ArangoDB of the respective release series:
- [x] 3.2 beta1
- [x] self-compiled devel branch

**Comment by danielhlarkin** [comment]
@mpv1989 If you change the action code to `throw new Error('My error context');` then we propagate the message upwards properly (`{"error":true,"code":500,"errorNum":500,"errorMessage":"My error context"}`).

Do we explicitly support handling raw string throws like this? If so, I can add a special case for strings, otherwise I would say we should leave it be and advise users to throw proper errors.

**Comment by jsteemann** [comment]
From my point of view both ways should work.
The "exception" attribute was populated in 3.1, and in devel it is not. If this was changed intentionally, then there should be something about it in the changelog, but I can't find it.
So we either need to fix it so it behaves as in 3.1 or make a note to the changelog.

**Comment by danielhlarkin** [comment]
We are not going to restore this behavior. Documentation has been added in PR #2595.
